server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -79,6 +80,7 @@ public class ExtensionsManager {
     public static final String REQUEST_EXTENSION_ENVIRONMENT_SETTINGS = "internal:discovery/enviornmentsettings";
     public static final String REQUEST_EXTENSION_ADD_SETTINGS_UPDATE_CONSUMER = "internal:discovery/addsettingsupdateconsumer";
     public static final String REQUEST_EXTENSION_UPDATE_SETTINGS = "internal:discovery/updatesettings";
+    public static final String REQUEST_EXTENSION_DEPENDENCY_INFORMATION = "internal:discovery/dependencyinformation";
```

Comment:
For future: We can change all the action name to `internal:extensions`. @mloufra can you create an issue to address this?

Reply:
sure

Reply:
Can you paste the issue link here?

Reply:
https://github.com/opensearch-project/opensearch-sdk-java/issues/305